### PR TITLE
Wallet/integrate flat transfer

### LINKED ISSUE #1762: Wallet - Use web3 to fetch wallet transactions
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story
As a developper, I want to to use web3 to fetch transactions so that Status does not depends on a centralized api (etherscan) for wallet transactions history.

### Description
[comment]: # (Feature or Bug? i.e Type: Bug)
_Type_: Feature

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
_Summary_: 
To reach MVP of the new wallet we used etherscan to fetch the transaction history of the wallet. This task will consist of implementing a new version of the transaction fetching logic using web3.

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
Transactions are fetched using web3

#### Actual behavior
[comment]: # (Describe what actually happened.)
Transactions are fetched using etherscan and are limited to the 20 last transactions

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
_Summary_: 

* [ ] implement and adapt this script in clojurescript https://ethereum.stackexchange.com/questions/2531/common-useful-javascript-snippets-for-geth/3478#3478 to get transactions for a given address from block X to Z in utils/transactions.cljs
* [ ] create an event that recursively calls the function for block n to n-step until the block when the wallet has been created is reached in ui.screens.wallet.events

Depending on the performance cost of this solution it might be worth it to keep the etherscan api call for the 20 first transactions, but use web3 as fallback in case it fails and fetch the rest of the transactions with web3.

### PR DESCRIPTION
**Pull request by yenda**
Fixes #1762

### Summary

- remove use of etherscan and subscriptions
- replace by status-go

#### Areas that maybe impacted
Wallet transaction history

Tested in emulator for  `satoshi document engage inflict goddess auction rule unfair bid next buddy shy` on ropsten

### Steps to test

status: ready

### CODE REVIEW

**Comment by yenda** [comment]
@rachelhamlin @asemiankevich for now we aimed at "good enough" transactions list, all the correct data is saved in status-go so we'll be able to settle and what informations we show and how we show it later without having to scrap the transactions already downloaded. This means that some transactions aren't shown as they were in previous versions, that is the case for erc721 token that are going to show up as incoming 0ERC20 transaction instead of previously outgoing ETH transactions, neither of which being a proper representation.

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ethereum/transactions/core.cljs [hunk 1]
```diff
@@ -198,6 +151,23 @@
...
 
+(re-frame/reg-fx
+ ::get-transfers
+ (fn [{:keys [chain-tokens from-block to-block]
+       :or {from-block "0"
+            to-block nil}}]
+   ;; start inbound token transaction subscriptions
+   ;; outbound token transactions are already caught in new blocks filter
+   (json-rpc/call
+    {:method "wallet_getTransfers"
```
Comment (flexsurfer) [inline src/status_im/ethereum/transactions/core.cljs:162]: move to constants?

Comment (yenda) [reply src/status_im/ethereum/transactions/core.cljs:162]: I don't think constants are usefull for this kind of things, also the call function only works with registered method

**Review by jeluard — APPROVED** [review]

**Comment by asemiankevich** [comment]
@yenda would you mind to squash your commits into 1 ?

**Comment by yenda** [comment]
@asemiankevich is it voluntarily separated in two, the first one starts using go service for live updates, the second one for history

**Comment by asemiankevich** [comment]
@yenda builds failed after last force push ((

**Comment by yenda** [comment]
@asemiankevich yes but don't worry about it, it is failing because of the last nix commit in develop, will get sorted out, but the old builds are still valid

**Comment by asemiankevich** [comment]
i've checked transaction history both on mainnet and ropsten on my devices , it looks good both new and recovered account. the only thing i noticed that transactions are slowly mined (especially tokens).. But anyway lets merge this PR @yenda

**Comment by yenda** [comment]
@asemiankevich yes it is slow for the first time but then it is persisted so should be fast

**Pull request merged**